src/main/kotlin/backcompat/ui/layout/MigLayoutBuilder.kt

```diff
@@ -281,9 +281,9 @@ private fun createComponentConstraints(constraints: Array<out CCFlags>? = null,
         cc().split = split
     }
 
-    if (growPolicy == GrowPolicy.SHORT_TEXT) {
-//    cc().minWidth("210")
-        cc().maxWidth("210")
+    when (growPolicy) {
+        GrowPolicy.SHORT_TEXT -> cc().maxWidth("210")
+        GrowPolicy.GROW_X -> cc().growX()
```

Comment:
This file is copy-pasted from the IDEA, and we'll get rid of it once we drop support for 2016.2, so it's better to handle this elsewhere.